src/test/scala/cromwell/MainSpec.scala

```diff
@@ -388,17 +390,16 @@ object MainSpec {
    * @return The return code of run.
    */
   def traceErrorWithExceptionRun(args: String*)(pattern: String, throwableClass: Class[_ <: Throwable] = classOf[Throwable]): Int = {
-    val workflowManagerSystem = new TestWorkflowManagerSystem
-    val result = waitForErrorWithException(pattern, throwableClass = throwableClass)(
-      printBlock("run", args) {
-        // Explicitly disable shutting down the actor system from within Main, there's a race condition to deliver
-        // log messages to the TestKit filter before the system is torn down.  Wait until we see the message we
-        // want, then shutdown the system ourselves.
-        new Main(enableTermination = false, () => workflowManagerSystem).run(args)
-      }
-    )(workflowManagerSystem.actorSystem)
-    TestKit.shutdownActorSystem(workflowManagerSystem.actorSystem, timeoutDuration)
-    result
+    withTestWorkflowManagerSystem { workflowManagerSystem =>
+      waitForErrorWithException(pattern, throwableClass = throwableClass)(
+        printBlock("run", args) {
+          // Explicitly disable shutting down the actor system from within Main, there's a race condition to deliver
+          // log messages to the TestKit filter before the system is torn down.  Wait until we see the message we
+          // want, then shutdown the system ourselves.
+          new Main(workflowManagerSystem).run(args)
```

Comment:
This comment might need to be reworded or removed.  This referred to passing the `enableTermination` flag explicitly, which is no longer required with your changes.